### PR TITLE
Explore: UI changes for split view and live tailing

### PR DESCRIPTION
**Pull request by ivanahuckova**
**What this PR does / why we need it**:

- [x]  Give the icons of the active toggle button state (orange) the brand gradient - `icon-brand-gradient` class has been created as it is used in different components across Explore.
- [x]  Give the icons of the active toggle button glow - `explore-active-button-glow` class has been created as it is used in different components across Explore.
- [x] Hides  “delete query” button when in live-tailing mode

Examples:
image
image
image
image

### CODE REVIEW

File: public/app/features/explore/ExploreToolbar.tsx [hunk 1]
```diff
@@ -278,15 +278,16 @@ export class UnConnectedExploreToolbar extends PureComponent<Props> {
...
-            <div className="explore-toolbar-content-item explore-icon-align">
-              <ResponsiveButton
-                splitted={splitted}
-                title="Clear All"
-                onClick={this.onClearAll}
-                iconClassName="fa fa-fw fa-trash icon-margin-right"
-                disabled={isLive}
-              />
-            </div>
+            {!isLive && (
```
Comment (ivanahuckova) [inline public/app/features/explore/ExploreToolbar.tsx]: Hide clear all button when live-tailing.

File: packages/grafana-ui/src/components/TimePicker/TimePicker.tsx [hunk 2]
```diff
@@ -11,24 +11,27 @@ import { TimePickerPopover } from './TimePickerPopover';
...
+    .lighten(20)
+    .toString();
   return {
     timePickerSynced: css`
       label: timePickerSynced;
       border-color: ${theme.colors.orangeDark};
       background-image: none;
       background-color: transparent;
       color: ${theme.colors.orangeDark};
+      box-shadow: 0px 0px 4px ${yellowLight};
```
Comment (ivanahuckova) [inline packages/grafana-ui/src/components/TimePicker/TimePicker.tsx]: Add glow according to [design](https://www.figma.com/file/I34UgAga9JvqjlxfKYZnve/Components?node-id=453%3A0).

File: public/sass/pages/_explore.scss [hunk 3]
```diff
@@ -6,6 +6,14 @@
   }
 }
 
+.icon-brand-gradient {
```
Comment (ivanahuckova) [inline public/sass/pages/_explore.scss]: As icon-brand-gradient is used at the multiple places across Explore, I have created class for it.

**Comment by jessover9000** [comment]
I love iiiiiiit 🔥🔥🔥 in Light theme it looks perfect! In dark theme I think the glow is a bit too bright. you could change the box-shadow color to #ff780a / the same orange tone as the border, that should work better :) other than that, well done!

**Comment by ivanahuckova** [comment]
@jessover9000 see below updated glow for dark theme:
image

File: public/sass/_variables.light.generated.scss [hunk 4]
```diff
@@ -183,6 +183,8 @@ $btn-drag-image: '../img/grab_light.svg';
 
 $navbar-btn-gicon-brightness: brightness(1.5);
 
+$btn-active-box-shadow: 0px 0px 4px rgba(234, 161, 51, 0.6);
```
Comment (ivanahuckova) [inline public/sass/_variables.light.generated.scss]: `$btn-active-box-shadow ` created for both theme as this box shadow is used in `explore-active-button-glow` that us used for all active-buttons glows (timesync, timepicker, live-tailing button)

File: packages/grafana-ui/src/themes/_variables.dark.scss.tmpl.ts [hunk 5]
```diff
@@ -187,6 +187,8 @@ $btn-drag-image: '../img/grab_dark.svg';
 
 $navbar-btn-gicon-brightness: brightness(0.5);
 
+$btn-active-box-shadow: 0px 0px 4px rgba(255,120,10,1);
```
Comment (ivanahuckova) [inline packages/grafana-ui/src/themes/_variables.dark.scss.tmpl.ts]: `$btn-active-box-shadow` has been created for both themes and is then used in `explore-active-button-glow` class. This class is then used to create active button glow for all buttons, that have active state - timepicker, time-sync button, live-tailing button.

File: public/app/features/explore/ExploreToolbar.tsx (hunk 1 above)
Comment (ivanahuckova) [inline public/app/features/explore/ExploreToolbar.tsx]: If user is in live-tailing state, `clear all queries` is not visible, as disabled state was not very clear.

File: public/sass/pages/_explore.scss [hunk 6]
```diff
@@ -14,6 +22,9 @@
   }
 }
 
+.explore-active-button-glow {
+  box-shadow: $btn-active-box-shadow;
+}
 .explore-ds-picker {
```
Comment (ivanahuckova) [inline public/sass/pages/_explore.scss]: `icon-brand-gradient` and `explore-active-button-glow` have been created as they are used in different components across explore.

**Comment by davkal** [comment]
Nice approach to making it more pop. Three issues:

- I find the gradient text and icon hard to read (I prefer no gradient there)
- after resuming Live, I have to click somewhere else before the button loses its active gray background (in light theme)
- i'd probably dial the glow a tiny bit more down. If you can "see it", it's too much. Can we make this more subtle?

File: public/app/features/explore/ExploreToolbar.tsx (hunk 1 above)
Comment (aocenas) [reply public/app/features/explore/ExploreToolbar.tsx]: Can we add a test for this? There are none for this component right now so no need to add more but would be good to start somewhere.

**Comment by ivanahuckova** [comment]
> I find the gradient text and icon hard to read (I prefer no gradient there)
@davkal Do you mean no gradient for the text (on other words, pulsing effect for text?).

> after resuming Live, I have to click somewhere else before the button loses its active gray background (in light theme)

✅Fixed

> i'd probably dial the glow a tiny bit more down. If you can "see it", it's too much. Can we make this more subtle?
@jessover9000 any suggestions? Currently it is ` 0px 0px 4px rgba(255,120,10,1)`. I have tried 
(and like) the same color with 2px instead of 4px - looks more subtle:
image

**Comment by marefr** [comment]
@ivanahuckova @davkal Would it be possible to get this reviewed and merged by tomorrow to be able to include it in release on Monday?

**Comment by jessover9000** [comment]
Ivana and I reworked the glow, but checking the gradient on the icons we think it works fine. On the text, there is no gradient. I checked contrasts there again and the only part where there is a slight issue is the very bottom part of the light theme, which doesn't impair the ability to recognize the icons all that much. So I think we can keep the gradient on the icons :)

**Review by jessover9000 — APPROVED** [review]

**Review by kaydelaney — APPROVED** [review]
lgtm

**Pull request merged**